src/eventstore/EventStoreSubscriber.cs

```diff
@@ -452,6 +453,13 @@ private void ProcessEvent(ResolvedEvent resolvedEvent)
                 var checkpointNumber = GetLastProcessedPosition(resolvedEvent);
 
                 WriteCheckpoint(checkpointNumber);
+                _logger.LogInformation(
+                    "Event dispatched from Eventstore subscriber ({0}/{1})",
+                    resolvedEvent.Event.EventStreamId,
+                    resolvedEvent.Event.EventNumber);
+#if DEBUG
+                _logger.LogTrace("Event dispatched {ResolvedEvent}]", resolvedEvent);
```

Comment:
I think including the actual event content is overkill - if you have event number / stream it is easy to go look up in the event store UI if needed.

Reply:
Removed logging of event content.